README.md

```diff
@@ -96,109 +55,57 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+We will be building our "Vet clinic" database during all the project. Here is the list of projects that will guide we through the steps described above.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Project 1: create animals table.]**
+- **[Project 2: query and update animals table.]**
+- **[Project 3: query multiple tables.]**
+- **[Project 4: add "join table" for visits.]**
+- **[Project 5: database performance audit.]**
+- **[Exercise: add database schema diagram.]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link- Soon](#)
 
```

Comment:
- [ ] It would be nice if you remove this Live Demo section since you don't have a live link for your project :thinking: Kindly remove it now to make your readme file descriptive and professional :+1: Once you will have your live link ready then you can add this Live Demo section with the live link in the readme file :blush:

Reply:
Thanks.